src/nih_wayfinding/app/scripts/mapping/map-control-service.js

```diff
@@ -55,6 +56,26 @@
                 .openOn(map);
             });
         }
+
+        var userMarker = null;
+        /**
+         * For a given point attach a marker on that point
+         *
+         * @param point {array} [Lng, Lat] array of coordinates
+         * @return undefined Shows (or moves) a marker
+         */
+        function trackUser(point) {
+            leafletData.getMap().then(function(map) {
+                var lnglat = [point[1], point[0]];
+                if (userMarker) {
+                  userMarker.setLatLng(lnglat);
+                  map.setZoom(19);
```

Comment:
Zoom should probably be set in the controller, so that this can be a bit  more generic. i.e. I can add the marker on the routing overview view.